styles.css

```diff
@@ -0,0 +1,2199 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;700&family=Roboto:wght@700&display=swap');
+
+@media only screen and (min-width: 768px) {
+  * {
+    max-width: 100%;
+    margin: 0;
+    padding: 0;
+  }
+
+  body {
+    width: 100%;
+    min-width: none;
+  }
```

Comment:
- [ ] In your design there is some huge space at the right side of the page kindly fix it. Check below :point_down:  the screenshot

![screencapture-file-media-aamir-Workplace-Microverse-Code-Reviews-Projects-desktop-portfolio-index-html-2023-03-05-03_19_04](https://user-images.githubusercontent.com/45226848/222931786-f9b6cfae-d9a3-4ffc-b9b3-46e85449c0a0.png)